src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown

```diff
@@ -50,14 +50,14 @@ If a Read request does not arrive within five minutes for a given reservation, t
 
 |Operation|Description|
```

Comment:
all below lines are linked to endpoint documentation and not schema, so no need to change links here..